stdlib/xml/dom/minicompat.pyi

```diff
@@ -1,3 +1,17 @@
-from typing import Any
+from typing import Any, Iterable, List, Tuple, Type, TypeVar
 
-def __getattr__(name: str) -> Any: ...  # incomplete
+_T = TypeVar("_T")
+
+StringTypes: Tuple[Type[str]]
+
+class NodeList(List[_T]):
+    length: int
+    def item(self, index: int) -> _T: ...
```

Comment:
```suggestion
    def item(self, index: int) -> Optional[_T]: ...
```
Returns None if the item is out of range

Reply:
Oh, so it does. PEP 8, will ya!